packages/datadog-plugin-next/test/next.config.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
 
     config.externals = {
       'react': `commonjs2 ${react}`,
-      'react-dom': `commonjs2 ${reactDom}`
+      'react-dom': `commonjs2 ${reactDom}`,
+      '@mapbox/node-pre-gyp': '@mapbox/node-pre-gyp'
```

Comment:
If this is added here, the correct version should also be added in `externals.json` to make sure it's installed for the plugin tests even if `pprof` is not present.

Reply:
What's `externals.json`? I'm not familiar with it.

Reply:
It's used to define additional dependencies that tests need for a given plugin, and installs that dependency besides each version that gets installed. So in this case, there are already `react` and `react-dom` defined here:
https://github.com/DataDog/dd-trace-js/blob/1e05a455db222864ea953a9aba8e932eecc49b43/packages/dd-trace/test/plugins/externals.json#L88-L97

Reply:
Fixed.